### DISCUSSION TITLE
BREAKING CHANGE - ESPHome update 2024.10.1 - FIXED in 10.2

### DISCUSSION THREAD

**Discussion by BigBobbas** [BigBobbas/ESP32-S3-Box3-Custom-ESPHome#100] [Announcements]
Updating to ESPHome version 2024.10.1 will cause the S3 Box to bootloop using this firmware. FIXED in 10.2

This looks to be an issue with the ADF Pipeline and a PR that was merged into this release. 
I am looking into this, if you have updated, and are using Home Assistant add-on. you can go to settings>> System>> backups and you should see a backup for your previous version. If you restore this backup, it will only restore ESPHome and not your full HA system.
Once restored - install the config to the device again from the ESPHome dashboard and everything should work as before.

I am looking into rectifying this issue 👍

**Comment by TheSerapher** [comment]
I guess this was fixed? Running ESPHome `2024.11.3` and not seeing boot loop issues 🙂

**Reply by BigBobbas** [reply]
yes, it was fixed in 2024.10.2 👍
